diff --git a/arch/arm/dts/imx6ul-14x14-evk.dtsi b/arch/arm/dts/imx6ul-14x14-evk.dtsi
index 463d7ca124..859723ed56 100644
--- a/arch/arm/dts/imx6ul-14x14-evk.dtsi
+++ b/arch/arm/dts/imx6ul-14x14-evk.dtsi
@@ -13,7 +13,7 @@
 
 	memory@80000000 {
 		device_type = "memory";
-		reg = <0x80000000 0x20000000>;
+		reg = <0x80000000 0x10000000>;
 	};
 
 	backlight_display: backlight-display {
@@ -65,7 +65,7 @@
 	};
 
 	panel {
-		compatible = "innolux,at043tn24";
+		compatible = "ontat,yx700wv03";
 		backlight = <&backlight_display>;
 
 		port {
